composer.json

```diff
@@ -29,7 +28,6 @@
         "phpcs": "phpcs -n --ignore=vendor --extensions=php .",
         "phpcbf": "phpcbf -n --ignore=vendor --extensions=php .",
         "phpstan": "phpstan analyse ./src ./tests --level=max --no-progress -c phpstan.neon",
-        "codeclimate": "test-reporter",
```

Comment:
Code Climate som vyhodil, kedze to nepouzivame, a nevedel som sa dostat ku tokenu.